android/clear_environment.sh

```diff
@@ -10,13 +13,35 @@ if [ ! -d android ]] ; then
     exit 1
 fi
 
-if [ "full" = "$1" ]; then
-    echo "full clean"
-    rm -rf 3rdParty/buildCache/android-tc
+if [ "android" = "$1" ]; then
+    echo "android clean"
+    rm -rf $cache_dir/android-tc
+elif [ "vcpkg" = "$1" ]; then
+    echo "vcpkg clean"
+    rm -rf $build_dir/vcpkg
+elif [ "ndk" = "$1" ]; then
+    echo "ndk clean"
+    rm -rf $build_dir/android-ndk-*
+elif [ "vcpkg_cache" = "$1" ]; then
+    echo "vcpkg_cache clean"
+    rm -rf $cache_dir/vcpkgcache/
+elif [ "cache" = "$1" ]; then
+    echo "cache dir clean"
+    rm -rf $cache_dir
+elif [ "build" = "$1" ]; then
+    echo "build dir clean"
+    rm -rf $cache_dir
```

Comment:
Should be `$build_dir`

Reply:
Yes, you are right. I will fix it. Thank for noticing.